hardware/eHouse/EhouseTcpClient.cpp

```diff
@@ -0,0 +1,482 @@
+/*
+ * File:   TcpClient.c
+ * ONLY for usage with eHouse system controllers | Intellectual property of iSys.Pl | Any other usage, copy etc. is forbidden 
+ * eHouse4Ethernet Server - described in UdpListener.c
+ * eHouse1 (RS-485) Server  //described in rs485.c
+ * eHouse1 under CommManager Supervision Server - described in UdpListener.c
+ * TCP Client for submitting events to Ethernet eHouse Controllers and eHouse 1 via CommManager in case of eHouse under CM supervision version
+ * 
+ * Author: Robert Jarzabek, iSys.Pl
+ * http://www.isys.pl/          Inteligentny Dom eHouse - strona producenta
+ * http://www.eHouse.pro/       Automatyka Budynku
+ * http://sterowanie.biz/       Sterowanie Domem, Budynkiem ...
+ * http://inteligentny-dom.ehouse.pro/  Inteligentny Dom eHouse Samodzielny montaż, zrób to sam, przykłady, projekty, programowanie
+ * http://home-automation.isys.pl/      eHouse Home automation producer home page
+ * http://home-automation.ehouse.pro/   eHouse Home Automation - Do it yourself, programming examples, designing
+ * Created on June 5, 2013, 10:00 AM
+ * Updated 2017-11-09
+ * globals.h - global configuration / main settings
+ * most important params 
+ * 
+ */
+
+//#include <iostream>
+#include "stdafx.h"
+#include "../main/Logger.h"
+#include<stdio.h> //printf
+#include <time.h>
+#include<string.h>    //strlen
+#include <sys/types.h>
+#include "../eHouseTCP.h"
+#ifndef WIN32
+#include <sys/socket.h>
```

Comment:
All these winsock includes (and in your other files) are already done in stdafx.h
Did you try to remove as much includes as possible already to see if you do not need them ?